app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    accuracy: ["Providers ask for proof of identification"],
+    notes: ["We need to collect year, but do we need to show it to providers?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Nationality",
+  items: {
+    question: "Nationality",
+    required: "Yes",
+    api: {
+      param: "candidate.nationality[]",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "may use this to assess if a candidate is [eligibile for student finance](https://www.gov.uk/student-finance/who-qualifies)"
+    }, {
+      who: ["hesa"],
+      what: "requires this “to respond to Parliamentary Questions, Written Questions, FOI requests on complex citizenship/nationality issues. To be able to perform analysis on trainee demographics at the start of an academic year.”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers ask for proof of identification"],
+    notes: ["We need to collect year, but do we need to show it to providers?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Second nationality",
+  items: {
+    question: "Add another nationality → Second nationality",
+    required: "No",
+    api: {
+      param: "candidate.nationality[]",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    notes: ["We needn’t ask for additional nationalities; dual nationals can enter their British or EEA-derived nationality"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "✘ Residency status",
+  items: {
+    question: "What is your residence status?",
+    required: "Yes (if country of nationality is not within the UK or EEA)",
+    api: {
+      param: "candidate.uk_residency_status",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "may use this field to assess if a candidate is [eligibile for student finance](https://www.gov.uk/student-finance/who-qualifies)"
+    }, {
+      who: ["hesa"],
+      what: "requires domicile to “distinguish between home and overseas students and to disaggregate students by origin. To obtain the numbers of students domiciled in particular EC countries and elsewhere abroad, and hence calculate participation rates. To forecast the demand for student allowances and grants. To record cross-border flows.”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Providers ask for documentation"],
+    notes: ["In addition to status, do we also need to ask for the date they entered the country? This might be relevant if an applicant has a Tier 5 Visa (from Australia or New Zealand, where people can work for up to 2 years in the UK). This may also impact funding decisions"]
```

Comment:
```suggestion
    notes: ["In addition to status, do we also need to ask for the date they entered the country? This might be relevant if an applicant has a Tier 5 Visa (from Australia or New Zealand, where people can work for up to 2 years in the UK). Date they entered the country can also impact funding."]
```